ETL/main.py

```diff
@@ -0,0 +1,196 @@
+import psycopg2
+import sys
+import os
+import json
+from airtable import Airtable
+import pandas
+from geopy.geocoders import Nominatim
+
+AIRTABLE_API_KEY = os.environ.get('AIRTABLE_API_KEY')
+AIRTABLE_BASE_ID = os.environ.get('AIRTABLE_BASE_ID')
+
+if AIRTABLE_API_KEY == None or AIRTABLE_API_KEY == '':
+    print('The Airtable API key is missing')
+    sys.exit(10)
+
+if AIRTABLE_BASE_ID == None or AIRTABLE_BASE_ID == '':
+    print('The Airtable Base ID is missing')
+    sys.exit(11)
+
+DATABASE_URL = os.environ.get('DATABASE_URL')
+
+if DATABASE_URL == None or DATABASE_URL == '':
+    if sys.argv[1] != None and sys.argv[1] != '':
+        DATABASE_URL = sys.argv[1]
+
+if DATABASE_URL == None or DATABASE_URL == '':
+    print('The postgres connection string DATABASE_URL is missing')
```

Comment:
Could be good to have a better message here like where to find it.

Reply:
Added more detail to the error message